src/pages/steps/LocationStep.tsx

```diff
@@ -284,6 +286,7 @@ const LocationStep = ({
                       };
                       field.onChange(updatedValue);
                       onChange(updatedValue);
+                      field.onBlur();
```

Comment:
why are these field.onBlurs necessary?

Reply:
In `Steps` we check if the field has a value true on is `isTouched`.

Description from docs:

**isTouched**: field has received a focus and blur event.

https://react-hook-form.com/api/useform/getfieldstate#main

It can probably also be done with dirty fields and checking `isDirty`. But since it was working this way already I kept that logic.

Reply:
👍